<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>728</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>728</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>721</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="RemoveBtn">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>442</y>
        <width>151</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
     <widget class="QComboBox" name="PreMenu">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>350</y>
        <width>242</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>د.</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>أ.</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>م.</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="NameList">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>12</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Listed Names</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="PreLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Pre</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="NameLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>393</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="IdLabel">
      <property name="geometry">
       <rect>
        <x>356</x>
        <y>352</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="EmailLabel">
      <property name="geometry">
       <rect>
        <x>356</x>
        <y>393</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="EmailBtn">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>443</y>
        <width>151</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CreateBtn">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>477</y>
        <width>151</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AddBtn">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>442</y>
        <width>151</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>42</y>
        <width>591</width>
        <height>281</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Pre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Email</string>
       </property>
      </column>
     </widget>
     <widget class="QLineEdit" name="IdLine">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>350</y>
        <width>242</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="NameLine">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>393</y>
        <width>242</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="EmailLine">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>393</y>
        <width>242</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="OMK3NDYBtn">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>450</y>
        <width>120</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>Click me :)</string>
      </property>
     </widget>
     <widget class="QLabel" name="OMK3NDYPic">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>331</width>
        <height>331</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="PageOneBtn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Certification Creation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PageTwoBtn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Something...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>728</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
